roles/openshift_node/defaults/main.yml

```diff
@@ -31,12 +31,9 @@ openshift_node_ami_prep_packages:
 - python-dbus
 - PyYAML
 - yum-utils
-- python2-boto
-- python2-boto3
```

Comment:
these make sense to remove

Reply:
These also aren't available in origin.

Centos decided to package these as 'python-boto'.  There is no package python-boto3 in Centos, at least not in the repos we're consuming.